build_for_ultimaker.sh

```diff
@@ -43,13 +43,94 @@ DEB_DIR=`pwd`/debian
 KCONFIG=`pwd`/configs/${BUILDCONFIG}_config
 KERNEL_BUILD=`pwd`/_build_armhf/${BUILDCONFIG}-linux
 
+INITRAMFS_COMPRESSION="${INITRAMFS_COMPRESSION:-.lzo}"
+INITRAMFS_ROOT_GID=${INITRAMFS_ROOT_GID:-0}
+INITRAMFS_ROOT_UID=${INITRAMFS_ROOT_UID:-0}
+INITRAMFS_SOURCE="${INITRAMFS_SOURCE:-initramfs/initramfs.lst}"
+INITRAMFS_IMG="${KERNEL_BUILD}/initramfs.cpio${INITRAMFS_COMPRESSION}"
+
+GEN_INIT_CPIO="${KERNEL_BUILD}/usr/gen_init_cpio"
+GEN_INITRAMFS_LIST="${KERNEL}/scripts/gen_initramfs_list.sh"
+
+BB_PKG="http://ftp.nl.debian.org/debian/pool/main/b/busybox/busybox-static_1.22.0-19+b3_armhf.deb"
```

Comment:
To prevent future errors, it would be better to copy this busybox package to the local build system as we do with all packages to make sure they a) exists and b) do not change which might lead to broken scripts expecting something that's no longer available.

Reply:
yes, i already have an issue where we want to compile our own u-boot instead, as we need different features than are available in the current debian version.